portfolio/src/main/webapp/index.html

```diff
@@ -9,17 +9,15 @@
   <body>
     <div id="content">
       <h1>Becca's Portfolio</h1>
-        <p>Welcome!</p>
-        <p>I'm a rising junior computer science major at Vanderbilt University</p>
-      <p>This is my portfolio.</p>
+      <p>Welcome!</p>
+      <p>I'm a rising junior computer science major at Vanderbilt University</p>
       <p>Click here to get a random fun fact about me:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
-
-        <div class="image-thumbnails">
-            <p> random pic for now </p>
-        <a href="images/earth.jpg"><img src="images/earth.jpg"/></a>
-      </div>
+      <button onclick="udpateRandomFact()">Randomize</button>
+      <div id="funFact-container"></div>
```

Comment:
For names of ids and classes, the convention we use is to separate each word with a hyphen and to not use uppercase.  So we'd want this id to be "fun-fact-container".  =)

Reply:
fixed!